cfgov/unprocessed/js/modules/FlyoutMenu.js

```diff
@@ -92,14 +114,17 @@ function FlyoutMenu( element ) {
       this.dispatchEvent( 'expandBegin', { target: this } );
       _isExpanded = true;
       _isAnimating = true;
-      // If transition is not supported, call handler directly (IE9/OperaMini).
-      if ( _transitionEndEvent ) {
-        _contentDom.addEventListener( _transitionEndEvent, _expandEndBinded );
+      if ( _expandTransitionMethod ) {
+        _expandTransitionMethod.call();
```

Comment:
Updated and added a comment in https://github.com/cfpb/cfgov-refresh/commit/1dcbb2a9ab7d759a71deebbe145689c1c395f2b1